cassandane/Cassandane/Unit/TestCase.pm

```diff
@@ -329,4 +329,120 @@ sub assert_num_lte
                   "$actual is not less-than-or-equal-to $expected");
 }
 
+{
+  package Cassandane::Test::URL;
```

Comment:
Often these little packages are so niche and just provide the underlying behaviour of something returned by a small function, so having them in their own file can be a bit much, but in this case I don't think it's a problem so I've moved it to Cassandane::Util::TestURL

Reply:
I guess I see that; on the other hand I find it deeply confusing when I'm in an unfamiliar file hunting around trying to solve a bug, or understand a behaviour, and I search for something like "^sub new" and end up reading entirely the wrong constructor, or whatever.

(This gets worse when there's a base class and subclasses of it defined in the same file, because then there's _lots_ of "subroutine with the same name but it's not the one you're looking for", but that's not the case here.)